consensus/service/src/consensus_service.rs

```diff
@@ -364,7 +364,7 @@ impl<
             .register_service(health_service)
             .register_service(attested_service)
             .register_service(build_info_service)
-            .bind_using_uri(&self.config.client_listen_uri);
+            .bind_using_uri_with_reloading(&self.config.client_listen_uri, self.logger.clone());
```

Comment:
Any reason not to just keep this as `bind_using_uri`? Is there a scenario where we want to disable explicit/triggered reloads?

Reply:
I'm happy to change `bind_using_uri` to always do this.